bot/exts/info/source.py

```diff
@@ -65,7 +65,7 @@ def get_source_link(self, source_item: SourceType) -> Tuple[str, str, Optional[i
 
         # Handle tag file location differently than others to avoid errors in some cases
         if not first_line_no:
-            file_location = Path(filename).relative_to("bot/")
+            file_location = str(filename)
```

Comment:
What was the point of this change?

Reply:
See this comment: https://github.com/python-discord/bot/issues/2022#issuecomment-997258029